### ISSUE TITLE
Submission right addition in Session Block not consistent with Contribution

### ISSUE THREAD

**Issue by jouvin** [indico/indico#3023]
When adding a presenter to a contribution, you have the option to give him the submission right clicking on the `Submitter` button. This is not possible when creating a session block and the result it that to allow the session manager to download materials you have to go through a lot of additional clicks to add it explicitely to the submitter list. It'd be nice to have the same behaviour as for contribution.

**Comment by ThiefMaster** [comment]
First of all, session *blocks* have no ACLs or materials - those are actually on the session itself.

For *downloading* you don't need submission rights - having access to the session is enough. If you mean uploading: This indeed requires at least session management rights. Allowing session *coordinators* to manage materials on the session could make sense though.

**Comment by jouvin** [comment]
@ThiefMaster you are completely right on everything! I created this issues after problems seen with an event whose timetable structure is a bit ugly, relying mainly on sub-contributions. And so the upload rights needed to be added to the contribution (see https://github.com/indico/indico/issues/2363). Giving the session coordinators the right to manage material, if this is not yet the case, would be very useful. Either always or, like for submission rights in the contribution, as an optional right that you could add from the session coordinator list, with a button similar to `Submitter`.

I let you decide if you want to close this issue or not...
